html/janus.js

```diff
@@ -1624,11 +1624,27 @@ function Janus(gatewayCallbacks) {
 		});
 	}
 
+	function stopAllTracks (stream) {
```

Comment:
Not sure why this has to be a private function. Can't this be sort of an object function that can be used in other contexts? Something like:

    Janus.stopAllTracks = function(stream) {

This way it can be used easily wherever it's needed, whether it's `janus.js` or the application itself.